newIDE/app/src/SceneEditor/index.js

```diff
@@ -521,7 +521,8 @@ export default class SceneEditor extends React.Component<Props, State> {
   ) => {
     this.instancesSelection.clearSelection();
     instances.forEach(instance =>
-      this.instancesSelection.selectInstance(instance)
+      this.instancesSelection.selectInstance(instance),
```

Comment:
This line is bugged ;)

Reply:
hahaha xD I should really stop coding late after work. It's actually surprising gdevelop didn't crash when I was testing it. That second parameter is no longer optional